overlord/snapstate/handlers.go

```diff
@@ -3706,7 +3706,7 @@ func (m *SnapManager) doCheckReRefresh(t *state.Task, tomb *tomb.Tomb) error {
 		return err
 	}
 
-	updated, tasksets, err := reRefreshUpdateMany(tomb.Context(nil), st, snaps, nil, re.UserID, reRefreshFilter, re.Flags, chg.ID())
+	updated, tasksetGroup, err := reRefreshUpdateMany(tomb.Context(nil), st, snaps, nil, re.UserID, reRefreshFilter, re.Flags, chg.ID())
```

Comment:
It can, I didn't create a pre-download change because at this point we can only re-refresh snaps that have just been refreshed so it would be very unlikely that the snap becomes busy in the meantime. But I guess there's no harm in doing that